docs/docs-content/clusters/cluster-management/backup-restore/enable-etcd-backup.md

```diff
@@ -0,0 +1,119 @@
+   | `cluster.config.etcd-snapshot-dir`           | Specifies the directory where the etcd snapshots are saved. By default, this value is `/var/lib/rke2/rancher/server/db/snapshots`.                                    |
+
+   For example,
+
+   ```yaml
+   cluster:
+     config:
+       etcd-snapshot-schedule-cron: 0 */1 * * *
+       etcd-snapshot-retention: 12
+       etcd-disable-snapshots: false
+       etcd-snapshot-dir: /var/lib/rancher/rke2/server/db/snapshots
+   ```
+
+    </TabItem>
+
+    <TabItem value="k3s" label="K3S">
+
+   Use the following parameters to configure scheduled backups.
+
+   | Parameter                                    | Description                                                                                                                                                           |
+   | -------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+   | `cluster.config.etcd-snapshot-schedule-cron` | A cron expression that controls the time and frequency of scheduled backups. For example, the default value `0 */1 * * *` means the scheduled backup runs every hour. |
+   | `cluster.config.etcd-snapshot-retention`     | The maximum number of etcd backups to retain.                                                                                                                         |
+   | `cluster.config.etcd-disable-snapshots`      | Controls whether or not to disable scheduled etcd snapshots.                                                                                                          |
+   | `cluster.config.etcd-snapshot-dir`           | Specifies the directory where the etcd snapshots are saved. By default, this value is `/var/lib/k3s/rancher/server/db/snapshots`.                                     |
+
+   For example,
+
+   ```yaml
+   cluster:
+     config:
+       etcd-snapshot-schedule-cron: 0 */1 * * *
+       etcd-snapshot-retention: 12
+       etcd-disable-snapshots: false
+       etcd-snapshot-dir: /var/lib/rancher/k3s/server/db/snapshots
+   ```
+
+    </TabItem>
+
+    </Tabs>
+
+## Validate
+
+To validate the snapshots are successfully configured, connect to the cluster via SSH and change into the directory
+where the etcd snapshots are saved. Confirm that the snapshots are being created in the directory.
+
+## Next Steps
+
+If your cluster experiences data corruption issues, you can use the etcd snapshots to restore the cluster to working
+conditions. Restoring a cluster is a complicated procedure that requires SSH access to the node and intimate knowledge
+of etcd. We strongly suggest you contact us for assistance at support@spectrocloud.com.
```

Comment:
```suggestion
conditions. Restoring a cluster may be a challenging procedure, depending on your experience and technical skill level. The restoration also requires SSH access to the node and intimate knowledge
of etcd. If you have concerns or need assistance, contact us [support@spectrocloud.com](mailto:support@spectrocloud.com) for additional guidance.
```
Lenny, I took a stab at rephrasing this to make it more clear that it "may be" challenging for some. I also think support would appriciate if we didn't come out of the gate stating "We strongly suggest ...." , but rather left the door open 😅